Make.inc

```diff
@@ -103,6 +104,12 @@ ifeq ($(OS), WINNT)
   shlibdir = $(libdir)
 endif
 
+ifeq ($(OS), FreeBSD)
```

Comment:
Oh, place it after line 15?
and... i try to understand your statement: "use it instead of the unconditional definition".
Would you explain more? I cannot follow your thinking right now.